contentpacks/utils.py

```diff
@@ -707,12 +707,20 @@ def remove_nonexistent_assessment_items_from_exercises(node_data: list, assessme
 
 def remove_exercises_without_assessment_items(node_data):
```

Comment:
Please rename this function to include the new functionality like removal of duplicate video entries.